make/wag.mk

```diff
@@ -74,8 +75,8 @@ endef
 # wag-generate-mod is a target for generating code from a swagger.yml using wag for modules repos
 # arg1: path to swagger.yml
 define wag-generate-mod
-@if [ -z "$$CI" ]; then \
-    bin/wag -output-path gen-go -js-path ./gen-js -file $(1); \
+@if [ -z "$$CI"]; then \
```

Comment:
iirc bash correctly the extra space in the conditional is required

Reply:
Interesting, I looked it up and you're correct. I guess os x cares less about that because it never errored. 

Changing.